care/facility/migrations/0483_patientnotes_and_notesedit.py

```diff
@@ -0,0 +1,73 @@
+# Generated by Django 5.1.4 on 2025-01-21 07:24
+
+import logging
+
+from django.db import migrations
+
+logger = logging.getLogger(__name__)
+
+MIGRATION_ID = 413380
+
+def migrate_patient_notes(apps, schema_editor):
+    logger.debug("Migrating Patient Notes")
+    PatientNotes = apps.get_model("facility", "PatientNotes")
+    NoteThread = apps.get_model("emr", "NoteThread")
+    NoteMessage = apps.get_model("emr", "NoteMessage")
+    DOCTORS = 10
+    for patient_note in PatientNotes.objects.all():
+        note_thread = NoteThread.objects.create(
+            title=patient_note.thread == DOCTORS and "Doctors Discussion" or "Nurses Discussion",
+            patient_id=patient_note.patient.migrated_emr_patient_id,
+            encounter_id=patient_note.consultation.migrated_emr_encounter_id if patient_note.consultation else None,
+            meta={
+                "migration_id": MIGRATION_ID,
+            },
+            created_date=patient_note.created_date,
+            modified_date=patient_note.modified_date,
+            created_by=patient_note.created_by,
+            updated_by=patient_note.created_by,
+        )
+        history = []
+        for edit in patient_note.edits.all():
+            history.append(
```

Comment:
previously we were creating patient note edits for all objects, we should skip creating the history object for such edits as the content is same